doc/en/_templates/style.html

```diff
@@ -0,0 +1,4 @@
+<style>
+    ul {list-style: none;}
+    li {margin: 5px 0};
+</style>
```

Comment:
```suggestion
    #features {width: 50%;}
</style>
```

(To widen the trainings & events sidebar.)